opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/ServiceContextProvider.java

```diff
@@ -92,22 +88,22 @@
             session.setId(serviceSessionId);
         }
 
-        KeyAndSaltDto keyAndSaltDto = getSessionSecretKey(request);
+        KeyWithParamsDto keyWithParamsDto = getSessionSecretKey(request);
```

Comment:
Dto in variable name is useless - `keyWithParams` is better

Reply:
ok